src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -33,6 +33,7 @@ class Yaml
     const PARSE_CONSTANT = 256;
     const PARSE_CUSTOM_TAGS = 512;
     const DUMP_EMPTY_ARRAY_AS_SEQUENCE = 1024;
+    const PARSE_FILE = 4096;
```

Comment:
2048

Reply:
nope, we have a deprecated flag below that uses 2048 :)